src/OpenTK/Platform/X11/X11DisplayDevice.cs

```diff
@@ -144,35 +161,44 @@ private bool QueryXinerama(List<DisplayDevice> devices)
                         dev.IsPrimary = true;
                         first = false;
                     }
```

Comment:
Leave a space after the closing brace. It sets what follows apart from the rest of the code.